common/beerocks/bwl/dwpal/ap_wlan_hal_dwpal.cpp

```diff
@@ -521,8 +521,9 @@ load_hostapd_config(const std::string &radio_iface_name, std::string &fname,
                     std::map<std::string, std::vector<std::string>> &hostapd_config_vaps)
 {
     bool loaded                                = false;
-    std::vector<std::string> hostapd_cfg_names = {"/var/run/hostapd-phy0.conf",
-                                                  "/var/run/hostapd-phy1.conf"};
+    std::vector<std::string> hostapd_cfg_names = {
+        "/var/run/hostapd-phy0.conf", "/var/run/hostapd-phy1.conf", "/var/run/hostapd-phy2.conf",
```

Comment:
It's not entirely clear yet why the phy are now named phy2 and phy3 instead.
Keeping phy0 and phy1 shouldn't hurt since prplMesh is going to try them all until it finds the corresponding one.